qgepqwat2ili/tests/test_qgep.py

```diff
@@ -148,11 +148,31 @@ def test_case_e_export_selection(self):
             ]
         )
         # Perform various checks
+        logger.warning("Perform various checks ... ")
+        # resultpath = os.path.join(tempfile.mkdtemp(), "export_VSA_KEK_2019_LV95.xtf")
+        #resultpath = os.path.join(tempfile.mkdtemp(), "export_SIA405_ABWASSER_2015_LV95.xtf")
+        logger.warning(path)
+        
+        
+        file1 = open(path, 'r')
+        Lines = file1.readlines()
+        count = 0
+        # Strips the newline character
+        for line in Lines:
+            count += 1
+            logger.warning("Line{}: {}".format(count, line.strip()))
+        
         root = ET.parse(path)
```

Comment:
@olivierdalang I have added some code to display the export.xtf
Now I see that each line is printed twice and I get only the SIA405_Abwasser_2015_LV95 version displayed.
Any hint how to check on both files and adjust the self.assertEqual calls to fit this?

- I would like to get two export files in the check also
- I would like to check both seperately
- When I manually export the same selection in the QGIS demo project I get these files:
![20230104_arbon_selection2](https://user-images.githubusercontent.com/1265865/210602572-77e30cac-2af3-4210-b3f0-6469a8758889.png)
![20230104_arbon_selection2_export_xtf_files_logs](https://user-images.githubusercontent.com/1265865/210602580-e9a70b76-7019-42b9-bdfb-911e60d72214.png)